### ISSUE TITLE
setup via gem install wagn fails due to missing seed

### ISSUE THREAD

**Issue by juffel** [wagn/wagn#432]
### short story:
Apparently, when installing the wagn gem the wagn/decko-seed is not fetched correctly. After adding the seed files manually to the directory of the installed card-gem, everything works fine!

### long story:
I'm using OS X 10.10 and tried to setup a wagn instance using ```$ gem install wagn```

and creating a new wagn project with ```$ wagn new blorg```

set up databaseconnection correctly and tried to seed this newly created database using ```$ wagn seed``` in the new blorg directory, which raised following error:

```
$ wagn seed
bundle exec rake wagn:seed
rake aborted!
Errno::ENOENT: No such file or directory - /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/card-1.15.5/db/seed/new/cards.yml
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/activerecord-3.2.16/lib/active_record/fixtures/file.rb:43:in `read'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/activerecord-3.2.16/lib/active_record/fixtures/file.rb:43:in `rows'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/activerecord-3.2.16/lib/active_record/fixtures/file.rb:29:in `each'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/activerecord-3.2.16/lib/active_record/fixtures.rb:670:in `block (2 levels) in read_fixture_files'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/activerecord-3.2.16/lib/active_record/fixtures/file.rb:20:in `open'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/activerecord-3.2.16/lib/active_record/fixtures.rb:669:in `block in read_fixture_files'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/activerecord-3.2.16/lib/active_record/fixtures.rb:668:in `each'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/activerecord-3.2.16/lib/active_record/fixtures.rb:668:in `read_fixture_files'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/activerecord-3.2.16/lib/active_record/fixtures.rb:548:in `initialize'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/activerecord-3.2.16/lib/active_record/fixtures.rb:482:in `new'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/activerecord-3.2.16/lib/active_record/fixtures.rb:482:in `block (2 levels) in create_fixtures'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/activerecord-3.2.16/lib/active_record/fixtures.rb:479:in `map'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/activerecord-3.2.16/lib/active_record/fixtures.rb:479:in `block in create_fixtures'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/activerecord-3.2.16/lib/active_record/connection_adapters/abstract_mysql_adapter.rb:232:in `disable_referential_integrity'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/activerecord-3.2.16/lib/active_record/fixtures.rb:476:in `create_fixtures'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/wagn-1.15.5/lib/wagn/tasks/wagn.rake:369:in `block (3 levels) in <top (required)>'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/wagn-1.15.5/lib/wagn/tasks/wagn.rake:67:in `block (2 levels) in <top (required)>'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/wagn-1.15.5/lib/wagn/tasks/wagn.rake:34:in `block (2 levels) in <top (required)>'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/bin/ruby_executable_hooks:15:in `eval'
/Users/julian/.rvm/gems/ruby-2.0.0-p353/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => wagn:bootstrap:load
(See full trace by running task with --trace)
dropping
creating
...
```

Trying to start the wagn server anyways:

```
$ wagn server
=> Booting WEBrick
=> Rails 3.2.16 application starting in production on http://0.0.0.0:3000
=> Call with -d to detach
=> Ctrl-C to shutdown server
Exiting
/Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/card-1.15.5/lib/card/codename.rb:68:in `const_missing': uninitialized constant Card::HtmlFormat (NameError)
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/wagn-1.15.5/rails/controllers/card_controller.rb:11:in `<class:CardController>'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/wagn-1.15.5/rails/controllers/card_controller.rb:8:in `<top (required)>'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/engine.rb:444:in `block (2 levels) in eager_load!'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/engine.rb:443:in `each'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/engine.rb:443:in `block in eager_load!'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/engine.rb:441:in `each'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/engine.rb:441:in `eager_load!'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/application/railties.rb:8:in `each'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/application/railties.rb:8:in `all'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/engine.rb:439:in `eager_load!'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/application/finisher.rb:53:in `block in <module:Finisher>'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/initializable.rb:30:in `instance_exec'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/initializable.rb:30:in `run'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/initializable.rb:55:in `block in run_initializers'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/initializable.rb:54:in `each'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/initializable.rb:54:in `run_initializers'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/application.rb:136:in `initialize!'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/railtie/configurable.rb:30:in `method_missing'
	from /Users/julian/Projects/wiki/blorg/config/environment.rb:5:in `<top (required)>'
	from /Users/julian/Projects/wiki/blorg/config.ru:3:in `block in <main>'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/rack-1.4.5/lib/rack/builder.rb:51:in `instance_eval'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/rack-1.4.5/lib/rack/builder.rb:51:in `initialize'
	from /Users/julian/Projects/wiki/blorg/config.ru:in `new'
	from /Users/julian/Projects/wiki/blorg/config.ru:in `<main>'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/rack-1.4.5/lib/rack/builder.rb:40:in `eval'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/rack-1.4.5/lib/rack/builder.rb:40:in `parse_file'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/rack-1.4.5/lib/rack/server.rb:200:in `app'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/commands/server.rb:46:in `app'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/rack-1.4.5/lib/rack/server.rb:304:in `wrapped_app'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/rack-1.4.5/lib/rack/server.rb:254:in `start'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/commands/server.rb:70:in `start'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/commands.rb:55:in `block in <top (required)>'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/commands.rb:50:in `tap'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.16/lib/rails/commands.rb:50:in `<top (required)>'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/wagn-1.15.5/lib/wagn/commands.rb:48:in `require'
	from /Users/julian/.rvm/gems/ruby-2.0.0-p353/gems/wagn-1.15.5/lib/wagn/commands.rb:48:in `<top (required)>'
	from script/wagn:6:in `require'
	from script/wagn:6:in `<main>'

```

According to the Error which was shown when trying to initially seed the wagn, there were some seed files missing. I found out, that in my card-gem directory those seeds were not included. After downloading them (wagn/card/db/seed) manually (from github project wagn/decko-seed) and adding them to the correct directory x/gems/card-1.15.5/db/seed a newly generated project works just fine.

I wonder why this seed initially wasn't included in gem installation.

**Comment by ethn** [comment]
Thanks for sending this!  The seed data is obviously supposed to be there; I'll try to get that fixed today and put out a new version.

We recently moved the seed data to a separate git submodule to make it easier to read code changes; something in the gem packaging process must have gotten goofed up in the process.

**Comment by ethn** [comment]
The good news is that this is fixed in 1.15.6.

The bad news is that I'm not entirely clear on why it was broken in 1.15.5.  The gemspec appears to have been correct.  Perhaps something got messed up with my submodule?

I'll have to come up with a mechanism to test this to ensure it doesn't break again, but you should be back in business.
